src/Backdrop.js

```diff
@@ -114,7 +135,7 @@ const Backdrop = ({
         ref={frontLayerRef}
       >
         <div className={classes.headerBox}>
-          <Typography variant='h6'>{headerText}</Typography>
+          {headerText}
           {isMobile && (
             <IconButton
               aria-label=''
```

Comment:
@tetogomez no problem having this empty?

Reply:
No, because the Backdrop component has default props